packages/arb-token-bridge-ui/src/components/TransferPanel/TokenApprovalDialog.tsx

```diff
@@ -70,12 +62,20 @@ export function TokenApprovalDialog(props: TokenApprovalDialogProps) {
     [estimatedGas, gasPrice]
   )
 
-  const approvalFeeText = useMemo(() => {
+  const ethFeeText = useMemo(() => {
     const eth = formatAmount(estimatedGasFees, { symbol: ether.symbol })
+    return eth
+  }, [estimatedGasFees])
+
+  const usdFeeText = useMemo(() => {
     const usd = formatUSD(ethToUSD(estimatedGasFees))
-    return `${eth}${isEthereumMainnet ? ` (${usd})` : ''}`
+    return `${isEthereumMainnet ? ` (${usd})` : ''}`
```

Comment:
nit: we can show this for any mainnet chain that uses eth as gas token, not just ethereum (we can do arb1 to something too)

Reply:
can leave this for later, and create a ticket

Reply:
created